packages/frontend/components/composed/article/ArticleMetadataBlock/ArticleMetadataBlock.tsx

```diff
@@ -8,16 +8,58 @@ import { ArticleMetadataBlockFragment$key } from "@/relay/ArticleMetadataBlockFr
 import MetadataFactory from "components/factories/MetadataFactory";
 import { MetadataProperty } from "components/layout";
 import { NamedLink, PrecisionDate } from "components/atomic";
+import ContributorName from "components/composed/contributor/ContributorName";
 
 export default function ArticleMetadataBlock({ data }: Props) {
   const entity = useMaybeFragment(fragment, data);
+  const authors = entity?.contributions?.edges?.filter(
+    ({ node }) => node.role?.toLowerCase() === "author"
+  );
```

Comment:
This doesn't seem optimal, but I wasn't sure how else to get just authors.

Reply:
This is fine for now - we may want some API change in the future, to get contributions by role